netjsonconfig/backends/openwrt/converters/zerotier.py

```diff
@@ -0,0 +1,27 @@
+from ...zerotier.converters import ZeroTier as BaseZeroTier
+from ..schema import schema
+from .base import OpenWrtConverter
+
+
+class ZeroTier(OpenWrtConverter, BaseZeroTier):
+    _uci_types = ['zerotier']
+    _schema = schema['properties']['zerotier']['items']
+
+    def __intermediate_vpn(self, vpn):
+        vpn.update(
+            {
+                '.name': self._get_uci_name(vpn.pop('name')),
+                '.type': 'zerotier',
+                'join': [vpn.pop('id')],
+                'enabled': not vpn.pop('disabled', False),
+            }
+        )
+        return super().__intermediate_vpn(vpn, remove=[''])
+
+    def __netjson_vpn(self, vpn):
+        vpn['id'] = str(vpn.pop('join')[0])
+        vpn['name'] = vpn.pop('.name').replace('_', '-')
+        # 'enabled' defaults to False in OpenWRT
+        vpn['disabled'] = vpn.pop('enabled', '0') == '0'
```

Comment:
What about a slightly more pythonic form?
```suggestion
        vpn['disabled'] = not vpn.pop('enabled', '0')
```

Reply:
I attempted this change, but the test fails because '0' is a string in this context.